src/exporter.ts

```diff
@@ -0,0 +1,118 @@
+import { ExportResult } from '@opentelemetry/base';
+import { ReadableSpan, SpanExporter } from '@opentelemetry/tracing';
+import { toSpan, toBuffer, createAuthProto } from './transform';
+import { PLATFORM, sendSpans } from './platform/index';
+import { generateLongUUID } from './utils';
+import { OTEL_VERSION, VERSION } from './version';
+import * as lsTypes from './types';
+import { Attributes } from './enums';
+
+const DEFAULT_SERVICE_NAME = 'otel-lightstep-exporter';
+const DEFAULT_SATELLITE_HOST = 'https://collector.lightstep.com';
+const DEFAULT_SATELLITE_PORT = '443';
+const DEFAULT_SATELLITE_PATH = '/api/v2/reports';
+const DEFAULT_URL = `${DEFAULT_SATELLITE_HOST}:${DEFAULT_SATELLITE_PORT}${DEFAULT_SATELLITE_PATH}`;
+
+/**
+ * Lightstep Exporter Config
+ */
+export interface LightstepExporterConfig {
+  serviceName?: string;
+  hostname?: string;
+  runtimeGUID?: string;
+  token: string;
+  url?: string;
```

Comment:
Can we take `collector_host`, `collector_port`, and `collector_path` instead of just the url for the satellite? The collector prefix will make these options align with lightstep-tracer-javascript, see: https://github.com/lightstep/lightstep-tracer-javascript#new-traceroptions.